bciers/apps/reporting/src/app/components/operations/Operations.tsx

```diff
@@ -0,0 +1,51 @@
+import { GridRowsProp } from "@mui/x-data-grid";
+
+import { actionHandler } from "@/app/utils/actions";
+import { getServerSession } from "next-auth";
+import { authOptions } from "@/app/api/auth/[...nextauth]/route";
+import { OperationRow } from "@reporting/src/app/components/operations/types";
+import ReportingOperationDataGrid from "./ReportingOperationDataGrid";
+
+export const formatOperationRows = (rows: GridRowsProp) => {
+  return rows.map(({ id, operator, name, bcghg_id }) => {
+    return {
+      id,
+      name,
+      bcghg_id,
+      operator: operator,
+    };
+  });
+};
+
+// 🛠️ Function to fetch operations
+export const fetchOperationsPageData = async () => {
+  try {
+    // fetch data from server
+    const pageData = await actionHandler(`registration/operations`, "GET", "");
+    return {
+      rows: formatOperationRows(pageData.data),
+      row_count: pageData.row_count,
+    };
+  } catch (error) {
+    throw error;
+  }
```

Comment:
I think the `try` block is not useful here if we spit the error right back out. Or is this some javascript weirdness that we try to avoid?